poetry/utils/env.py

```diff
@@ -879,9 +879,13 @@ class Env(object):
 
     def __init__(self, path: Path, base: Optional[Path] = None) -> None:
         self._is_windows = sys.platform == "win32"
+        self._is_mingw = sysconfig.get_platform() == "mingw"
```

Comment:
```suggestion
        self._is_mingw = sys.platform == "msys"
```
This should work here, right? Would be good to keep the usage consistent. I am not too familiar with using mingw, would be good to get a confirmation.

Reply:
@abn Thanks for the review, I really appreciate it!

No that won't work:
```
$ python
Python 3.8.8 (default, Feb 20 2021, 07:16:03)  [GCC 10.2.0 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> print(sys.platform)
win32
```